sms/v1/openapi.yaml

```diff
@@ -81,6 +100,146 @@ paths:
+            type: string
+        - name: from
+          in: query
+          description: |
+            This parameter gives the identification of the sending party, which can either be the phone number in international format or an alphanumeric identifier with up to 11 characters.
+
+            `Some destination networks impose restrictions on the sender ID format. Please check the coverage list
+            and contact us for more information.`
+          required: true
+          schema:
+            type: string
+        - name: message
+          in: query
+          description: |
+            The message will be sent with characters encoded either in GSM standard alphabet (GSM-7) or GSM Unicode alphabet (UCS-2).
+
+            - GSM standard alphabet [GSM-7](http://en.wikipedia.org/wiki/GSM_03.38). Maximum length is 160 characters per single message and 153 characters
+            per concatenated message.
+            - GSM Unicode alphabet [UCS-2](http://en.wikipedia.org/wiki/GSM_03.38). Maximum length is 70 characters per single message and 66 characters
+            per concatenated message.
+
+            tyntec automatically splits the sent message into concatenated messages if the message
+            exceeds the given limits. These concatenated messages are unified by the handset once again and
+            they are displayed as one message (if supported by the handset).
+
+            `tyntec will charge for each part of the concatenated message as an individual message.`
+
+          required: true
+          schema:
+            type: string
+        - name: encoding
+          in: query
+          description: Encoding selection between GSM7, UNICODE or the default AUTO selection
+          schema:
+            type: string
+            enum:
+              - AUTO
+              - GSM7
+              - UNICODE
+            default: AUTO
+        - name: gateway
+          in: query
+          description: Which gateway to use for the delivery of the message
+          schema:
+            type: string
+            default: DEFAULT
+        - name: conversion
+          in: query
+          description: Is this message subject to conversion ratio
+          schema:
+            type: boolean
+            default: false
+        - name: sendDateTime
+          in: query
+          description: >-
+            Any future date in the format “yyyy-MM-ddT-HH:mm:ss+HH:mm” [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601).
+            If not set, message will be sent immediately.
+            The default time zone is [UTC](https://en.wikipedia.org/wiki/Coordinated_Universal_Time)
+          schema:
+            type: string
+        - name: validity
+          in: query
+          description: Validity in minutes for this message
+          schema:
+            type: string
+            default: '2880'
+        - name: reference
+          in: query
+          description: A custom reference that will mark the Delivery Report
```

Comment:
Isn't this more "A customer reference attached to the Delivery Reports"?

Reply:
If you say so!! I believe you!!!